### PR TITLE
chore(deps): update

### PR DESCRIPTION
**Pull request by evilebottnawi**
update parser

### CODE REVIEW

File: tests/preserve_line/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -527,9 +527,10 @@ $foo->bar->baz
     ->doOneMoreThing($config)
     ->bar->run(function () {
         $console->log('Bar');
-    }); // Run this
+    });
 
 // Hello world
+// Run this
```
Comment (evilebottnawi) [inline tests/preserve_line/__snapshots__/jsfmt.spec.js.snap]: Expected, before we have invalid position for this comment

File: src/printer.js [hunk 2]
```diff
@@ -1038,8 +1038,8 @@ function printLines(path, options, print, childrenAttribute = "children") {
         between && between[1] && between[1].includes("\n")
           ? hardline
           : wrappedParts.length > 0
-            ? " "
-            : "";
+          ? " "
+          : "";
```
Comment (evilebottnawi) [inline src/printer.js]: Prettier change how ternary output

**Comment by evilebottnawi** [comment]
What the hell with `prettier`

**Comment by evilebottnawi** [comment]
I can't believe :confused:  how it is happens? Locally all fine

File: package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "eslint-plugin-prettier": "3.0.0",
     "jest": "23.6.0",
     "jest-runner-eslint": "0.6.0",
-    "prettier": "prettier/prettier"
+    "prettier": "prettier/prettier#f6d8be881c6b34235da7c13cfe072727aa71fed8"
```
Comment (evilebottnawi) [inline package.json]: Temporary solution to avoid problem with npm/yarn cache and git repo

**Comment by evilebottnawi** [comment]
I hate travis, why need use own cache npm register :anger:

**Comment by evilebottnawi** [comment]
Can't be solved in our side, merge as is to continue development, hope problem will disappear

**Comment by czosel** [comment]
I could also reproduce the failing tests locally, are you sure it's not something in your local env?

**Pull request merged**